compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -1021,12 +1021,9 @@ fn link_natively(
                         && (code < 1000 || code > 9999)
                     {
                         let is_vs_installed = windows_registry::find_vs_version().is_ok();
-                        // FIXME(cc-rs#1265) pass only target arch to find_tool()
-                        let has_linker = windows_registry::find_tool(
-                            sess.opts.target_triple.tuple(),
-                            "link.exe",
-                        )
-                        .is_some();
+                        let has_linker =
+                            windows_registry::find_tool(sess.opts.target.arch, "link.exe")
```

Comment:
Should be `sess.target`, I think.

Reply:
That's what I get for forgetting to run `./x.py check`. :sweat_smile: